scripts/manage/landing_page_base.md

```diff
@@ -167,6 +167,12 @@ Next, apply this change to the Statefulset managing the Halyard daemon:
 ~/spinnaker-for-gcp/scripts/manage/update_halyard_daemon.sh
 ```
 
+### Upgrade Management Console
```

Comment:
There's a section "Update the console" further up that does something quite different. Perhaps add a description here to explain?

/cc @dorbin

Reply:
Done.